### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by dvdfu**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import sys
+
 from parser import Parser
 from router import Router
 from timeline import Timeline
 
-
```
Comment (davidozhang) [inline main.py:6]: we need the extra space for pep-8. do you have the linter installed?

File: git_runner.py [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import os
 
-
```
Comment (davidozhang) [inline git_runner.py:3]: ditto

File: parser.py [hunk 3]
```diff
@@ -11,18 +11,15 @@ class ParserException(GitUndoException):
 
     def __init__(self):
         self.parser = argparse.ArgumentParser(
-            description=(
-                'Wrapper utility around Git to provide undo/redo functions.'
-            ),
+            description = ('Wrapper utility around Git to provide undo/redo functions.'),
```
Comment (davidozhang) [inline parser.py]: im pretty sure this will break the 79-character limit per line for pep-8.

File: timeline.py [hunk 4]
```diff
@@ -1,10 +1,8 @@
 import fileinput
 
-
```
Comment (davidozhang) [inline timeline.py:3]: ditto

File: undo_runner.py [hunk 5]
```diff
@@ -3,23 +3,20 @@
 from commands.add import AddCommand
 from commands.commit import CommitCommand
 
-
```
Comment (davidozhang) [inline undo_runner.py:6]: ditto

File: timeline.py [hunk 6]
```diff
@@ -29,7 +27,7 @@ def get_last_line(self):
     # Pop the last line from
     def pop_last_line(self):
         prev_line = None
-        for line in fileinput.input(self.path, inplace=True):
+        for line in fileinput.input(self.path, inplace = True):
```
Comment (davidozhang) [inline timeline.py]: in python you don't need to space out the operators

Comment (dvdfu) [reply timeline.py]: we do this inconsistently, though, so I made it more consistent

**Review by ldreyes — APPROVED** [review]

**Pull request merged**